openstack/cce/v1/nodes/get.go

```diff
@@ -1,19 +1,25 @@
 package nodes
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack"
 )
 
-type ListNodes struct {
-	Kind       string       `json:"kind"`
-	ApiVersion string       `json:"apiVersion"`
-	Metadata   MetadataLink `json:"metadata"`
-	Nodes      []Node       `json:"items"`
-}
+// Get retrieves a particular nodes based on its unique ID and cluster ID.
+func Get(client *golangsdk.ServiceClient, clusterID, k8sName string) (*GetNode, error) {
+	raw, err := client.Get(fmt.Sprintf("https://%s.%s", clusterID, client.ResourceBaseURL()[8:])+
```

Comment:
Base URL for all methods should be moved to a function (I can see such in previous implementation), there is too much duplication right now.

In current implementation joining slice of predefined strings makes little sense and makes code less readable.